__tests__/services/service.spec.ts

```diff
@@ -0,0 +1,149 @@
+import {
+    Status,
+    RemoteDataSuccess,
+    RemoteDataFailure
+} from '../../libs/remoteData';
+
+import { 
+    service,
+    applyDataTransformer,
+    applyErrorTransformer,
+    resolveServiceMap,
+    PromiseRemoteDataResultMap
+} from '../../services/service';
+
+describe('Service `service`', () => {
+
+    interface TransformedData<T> {
+        transformed: T;
+        isTransformed: true;
+    }
+
+    const responseDataFailed = <RemoteDataFailure<string>> {
+        error: 'error',
+        status: Status.Failure
+    }
+
+    const responseDataSuccess = <RemoteDataSuccess<string>> {
```

Comment:
The same issue as above, please use https://github.com/beda-software/aidbox-react/blob/master/libs/remoteData.ts#L29